R/source.R

```diff
@@ -255,39 +262,37 @@ invoke_cargo <- function(toolchain, specific_target, dir, profile,
   }
 }
 
-generate_cargo.toml <- function(libname = "rextendr", dependencies = NULL, patch.crates_io = NULL,
-                                extendr_version = "*", extendr_macros_version = extendr_version) {
-  cargo.toml <- c(
-    '[package]',
-    glue::glue('name = "{libname}"'),
-    'version = "0.0.1"\nedition = "2018"',
-    '[lib]\ncrate-type = ["cdylib"]',
-    '[dependencies]',
-    glue::glue('extendr-api = "{extendr_version}"'),
-    glue::glue('extendr-macros = "{extendr_macros_version}"')
-  )
-
-  # add user-provided dependencies
-  cargo.toml <- c(cargo.toml, dependencies)
-
-  # add user-provided patch.crates-io statements
-  cargo.toml <- c(
-    cargo.toml,
-    "[patch.crates-io]",
-    patch.crates_io
+generate_cargo.toml <- function(
+                                libname = "rextendr",
```

Comment:
The first argument should be on the same line as `function(`. Does styler write it like this?